lib/jsonapi/utils.rb

```diff
@@ -245,7 +248,7 @@ def count_records(records, options)
 
     def setup_request
       @request ||=
-        JSONAPI::Request.new(
+        JSONAPI::RequestParser.new(
```

Comment:
Well, JU's releases get based only on ["stable" releases from JR](https://github.com/b2beauty/jsonapi-utils/blob/master/jsonapi-utils.gemspec#L22). I appreciate the warning, but for now we could only maintain a branch with regard to this and other JR's breaking changes for a future release. Thoughts?